src/Contracts/ViewableService.php

```diff
@@ -49,7 +49,7 @@ public function getUniqueViewsCount($viewable, $period = null): int;
      * @param  \Illuminate\Database\Eloquent\Model  $viewable
      * @return bool
      */
-    public function addViewTo($viewable): bool;
+    public function addViewTo($viewable, $ip = ''): bool;
```

Comment:
While I don't have any plan for chaning this methods signature in `v2.x`, I think it's better to replace `$ip = ''` with `$properties = []` or `$data  = []`, so if we want to add more options, they can pass it inside the same array instead of `...->addViewTo($post, false, $someVar, $someOtherVar)`.